chart/values.yaml

```
@@ -586,6 +586,9 @@ workers:
 
   podAnnotations: {}
 
+  # Labels specific to workers
```

```suggestion
  # Labels specific to workers objects and pods
```

Should be like this across the board, also in the schema description.

---

done